rendercv/data/models/rendercv_settings.py

```diff
@@ -0,0 +1,134 @@
+"""
+The `rendercv.models.rendercv_settings` module contains the data model of the
+`rendercv_settings` field of the input file.
+"""
+
+from typing import Optional
+
+import pydantic
+
+
+class RenderCVSettings(pydantic.BaseModel):
+    """This class is the data model of the rendercv settings. The values of each field
+    updates the `rendercv_settings` dictionary.
+    """
+
+    model_config = pydantic.ConfigDict(
```

Comment:
Never use `model_config` and always inherit from `base.py`.

Reply:
Done.